layers/convolutional.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1, padding=0,
         self.prior_std_z = prior_std_z
         self.use_bias = False
         self.dof = dof
-        self.mean_w = Parameter(torch.Tensor(out_channels, in_channels // groups, *self.kernel_size))
+        self.mean_w = Parameter()
```

Comment:
Don't you need to specify the shape of the parameter here?

Reply:
Maybe? I made this change because you had made this change on your other branch. I can revert it.